src/Modules/ModParser.lua

```diff
@@ -920,6 +920,7 @@ local preFlagList = {
 	["^enemies affected by your spider's webs [th]a[vk]e "] = { tag = { type = "MultiplierThreshold", var = "Spider's WebStack", threshold = 1 }, applyToEnemy = true },
 	["^enemies you curse take "] = { tag = { type = "Condition", var = "Cursed" }, applyToEnemy = true, modSuffix = "Taken" },
 	["^enemies you curse have "] = { tag = { type = "Condition", var = "Cursed" }, applyToEnemy = true },
+	["^enemies you curse are "] = { tag = { type = "Condition", var = "Cursed" }, applyToEnemy = true },
```

Comment:
For the parser to pick up the right modForm from the formList, it'll have to exclude the "are", since it's in the formList: ```["^are "] = "FLAG",```

```suggestion
	["^enemies you curse "] = { tag = { type = "Condition", var = "Cursed" }, applyToEnemy = true },
```